src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -357,6 +362,35 @@ export interface TiffLocation {
   bands: string[];
 }
 
+/**
```

Comment:
An example and/or unit test might be helpful

Reply:
added unit tests